src/api/routes/channels/#channel_id/messages/index.ts

```diff
@@ -207,6 +207,19 @@ router.post(
 			);
 		}
 
+		if (channel.isThread()) {
+			channel.message_count = (channel.message_count || 0) + 1;
+			channel.total_message_sent = (channel.total_message_sent || 0) + 1;
+			Promise.all([
+				channel.save(),
+				emitEvent({
+					event: "CHANNEL_UPDATE",
+					data: { ...channel, newly_created: false },
+					guild_id: channel.guild_id
+				})
+			]);
+		}
```

Comment:
we need the same code in the PUT method too (src/api/routes/channels/#channel_id/messages/#message_id/index.ts)